qa/tasks/scripts/ses_install_rgw_part1.sh

```diff
@@ -12,12 +12,12 @@ rgw_configurations:
 echo "$rgw_sls" | sed '/^$/d' >> /srv/pillar/ceph/rgw.sls
 
 random_minion_fqdn=${storage_minions[0]}
-random_minion=`echo $random_minion_fqdn | cut -d . -f 1`
+random_minion=$(echo $random_minion_fqdn | cut -d . -f 1)
 
 echo "role-rgw/cluster/${random_minion_fqdn}.sls" >> /srv/pillar/ceph/proposals/policy.cfg
 
 random_minion2_fqdn=${storage_minions[1]}
 
-random_minion2=`echo $random_minion2_fqdn | cut -d . -f 1`
+random_minoin2=${random_minion2_fqdn%.*}
```

Comment:
it should be two % since domain name can look like, `name.subdomain.domain.zone`

Reply:
The same for the rest of the code.

Reply:
```
       ${parameter%word}
       ${parameter%%word}
              Remove  matching  suffix  pattern.  The word is expanded to produce a pattern just as in pathname expansion, and matched against the expanded
              value of parameter using the rules described under Pattern Matching below.  If the pattern matches a trailing portion of the  expanded  value
              of  parameter, then the result of the expansion is the expanded value of parameter with the shortest matching pattern (the ``%'' case) or the
              longest matching pattern (the ``%%'' case) deleted.  If parameter is @ or *, the pattern removal operation is applied to each positional  pa-
              rameter in turn, and the expansion is the resultant list.  If parameter is an array variable subscripted with @ or *, the pattern removal op-
              eration is applied to each member of the array in turn, and the expansion is the resultant list.
```

Reply:
> it should be two % since domain name can look like, `name.subdomain.domain.zone`

it's done for our teuthology tests.

Reply:
what do you  mean?

Reply:
For example, when you run it in our teuthology tests the host name looks like `target-ses-064.ecp.suse.de` and your code will fail.

Reply:
it's done kyr